src/Console/Application.php

```diff
@@ -119,24 +117,13 @@ public function run(InputInterface $input = null, OutputInterface $output = null
         return parent::run($input, $output);
     }
 
-    public function doRun(InputInterface $input, OutputInterface $output)
+    public function doRun(InputInterface $input, OutputInterface $output): int
     {
-        $output->writeln(self::LOGO);
+        $output->writeln([self::LOGO, $this->getLongVersion(), "\n"]);
```

Comment:
why do we need `"\n"` here? it adds too much space before the next non-empty line: https://travis-ci.org/infection/infection/jobs/599743124#L810-L812

I would say it's not needed or just `''`

Reply:
It does in this case, but that's due to what comes after emitting a bunch of blank lines, rather than the fault of this logic. When I ran Infection locally, the order of output was not consistent: it alternated between two variants, and depending on exactly what was output, the blank line was either needed to separate the logo from the following section or it wasn't because the subsequent section emitted a bunch of blank lines as its preamble. Suffice it to say, I don't know what governs the output order or why it's not always consistent, but this problem may be better addressed elsewhere.

Reply:
Specifically, the prompt asking *"Which source directories do you want to include (comma separated)?:"* would sometimes appear before any other output. In any case, I've removed the line break as requested.

Reply:
this is very strange :| Feel free to open a separate issue if this is consistently reproduced